File: eCodes.Services/Database/Notification.cs
using System;
using System.Collections.Generic;

namespace eCodes.Services.Database
{
    public partial class Notification
    {
        public int NotificationId { get; set; }
        public string NotificationDesc { get; set; } = null!;
        public DateTime NotificationDateTime { get; set; }
        public int BuyerId { get; set; }

        public virtual Buyer Buyer { get; set; } = null!;
    }
}

File: ecodes_mobile/lib/model/notification.dart
import 'package:json_annotation/json_annotation.dart';



part 'notification.g.dart';

@JsonSerializable()
class Notif{
     int? notificationId;
     String? notificationDesc; 
     DateTime? notificationDateTime; 
     int? buyerId;

  Notif(){

  }

  factory Notif.fromJson(Map<String, dynamic> json) => _$NotifFromJson(json);

  /// Connect the generated [_$NotifToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$NotifToJson(this);


}